src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css

```
@@ -437,14 +437,18 @@
 .monaco-workbench .part.editor > .content .editor-group-container > .title .editor-actions {
 	cursor: default;
 	flex: initial;
-	padding: 0 8px 0 4px;
+	padding: 0 8px 0 8px;
 	height: 35px;
 }
 
 .monaco-workbench .part.editor > .content .editor-group-container > .title .editor-actions .action-item {
```

You can remove the next line below and combine it here like:
```suggestion
.monaco-workbench .part.editor > .content .editor-group-container > .title .editor-actions .action-item:not(:last-child) {
```